web/package.json

```diff
@@ -122,7 +122,7 @@
     "postcss-import": "^11.1.0",
     "postcss-loader": "^2.1.6",
     "postcss-url": "^7.3.2",
-    "prettier": "^1.15.1",
+    "prettier": "^1.16.3",
```

Comment:
It seems all the prettier editor plugins were updated to point to a particular parser or something or other, and now they all require prettier ^1.16?  Anyway, this made prettier work in my editor again, hopefully anyone else on our team using a prettier plugin won't be hurt by this change...  If so, let's chat and find a solution!